### ISSUE TITLE
Unable to set Store opening time/closing time through vendor

### ISSUE THREAD

**Issue by ManigandanRaamanathan** [weDevsOfficial/dokan#291]
Unable to set Store opening time/closing time through vendor. And how to display the opening and closing time in the rest api as well?

**Comment by saimonh3** [comment]
@ManigandanRaamanathan This feature is not available in dokan yet.

**Comment by ManigandanRaamanathan** [comment]
@saimonh3 May I know why it is not available yet? Since the codes are fully done by the dev team but havent merge yet. I tried following the modification based on the codes that the dev team performed, and it actually works after following the modification. 

But the entire custom code erased whenever updating to latest dokan version. I had to keep on replicating all the changes in codes manually whenever you release a new update without the time code merge.
